chat.tox.qTox.json

```diff
@@ -0,0 +1,44 @@
+{
+	"app-id": "chat.tox.qTox",
```

Comment:
Seems like a bad app-id as it isn't directly associated with `tox.chat`? They have their own website and own repos.

Reply:
Upstream also should rename their desktop and appdata files as per the spec to also be an app-id like this.

Until then the package can use `rename-desktop-file`, `rename-appdata-file`, `rename-icon` properties.

Reply:
What would be a good app-id for example?

Reply:
The domain used is https://qtox.github.io/

So `io.github.qtox.qTox` would make the most sense to me.

Reply:
Created a PR to change the app id https://github.com/qTox/qTox/pull/5137